src/components/ui/ImageUpload.tsx

```diff
@@ -40,14 +54,43 @@ class ImageUpload extends Component<Props> {
     field.set('');
   }
 
+  onDropRejected(rejectedFiles) {
+    for (const file of rejectedFiles) {
+      for (const error of file.errors) {
+        if (error.code === 'file-too-large') {
+          this.setState({ errorState: true });
+          this.setState(
+            (this.errorMessage = {
+              message: this.props.textMaxFileSizeError,
+            }),
+          );
+        }
+      }
+    }
+  }
+
   render() {
-    const { field, className, multiple, textDelete, textUpload } = this.props;
+    const {
+      field,
+      className,
+      multiple,
+      textDelete,
+      textUpload,
+      textMaxFileSize,
+      maxSize,
+      maxFiles,
+    } = this.props;
 
     const cssClasses = classnames({
       'image-upload__dropzone': true,
       [`${className}`]: className,
     });
 
+    const maxSizeParse =
```

Comment:
```suggestion
    const maxSizeParse: number =
```

This will make extra sure that maxSize is a number as well